### PR TITLE
Lots of NCRC HowTo TLC

### LINKED ISSUE #22444: Documentation on running tests for installed binaries is out of date/inaccurate
## Bug Description
Take the following examples: https://mooseframework.inl.gov/ncrc/ncrc_local_binary.html
We are instructing folks to run tests by way of `bison-opt --tests`. This has changed to first requiring the user to copy the tests/spec files into a usable location.

## Steps to Reproduce
Our instructions being wrong is clear enough evidence.

## Impact
Emails are being generated asking for assistance, when they should not have to be

### PR DESCRIPTION
**Pull request by milljm**
More than just being incorrect, there were plenty of areas that needed some TLC...

Closes #22444

### CODE REVIEW

File: modules/doc/content/ncrc/ncrc_local_binary.md [hunk 1]
```diff
@@ -42,144 +42,176 @@ optional arguments:
...
+##### List all available NCRC applications
 
--  List all available NCRC applications
+```bash
+ $> ncrc list
 
-  ```bash
-  $> ncrc list
+# Use 'ncrc search name-of-application' to list more detail
+# NCRC applications available:
```
Comment (cticenhour) [inline modules/doc/content/ncrc/ncrc_local_binary.md:53]: Did you mean for this to be the highest level of heading?

File: modules/doc/content/ncrc/ncrc_local_binary.md [hunk 2]
```diff
@@ -42,144 +42,176 @@ optional arguments:
...
-  ncrc-griffin              2021_07_29         build_0  ncrc-applications
-  ncrc-griffin              2021_08_27         build_0  ncrc-applications
-  ncrc-griffin              2021_08_31         build_0  ncrc-applications
-  ncrc-griffin              2021_09_03         build_0  ncrc-applications
-  ncrc-griffin              2021_09_09         build_0  ncrc-applications
-  ncrc-griffin              2021_09_15         build_0  ncrc-applications
-  ncrc-griffin              2021_09_25         build_0  ncrc-applications
-  ncrc-griffin              2021_10_02         build_0  ncrc-applications
-  ```
+Installing an NCRC application requires authentication. If you have sufficient privileges the package installation will commence:
```
Comment (cticenhour) [inline modules/doc/content/ncrc/ncrc_local_binary.md:81]:
```suggestion
Installing an NCRC application requires authentication. If you have sufficient privileges, the package installation will commence:
```

File: modules/doc/content/ncrc/ncrc_local_binary.md [hunk 3]
```diff
@@ -42,144 +42,176 @@ optional arguments:
...
+```
 
-  Username: johndoe
-  PIN+TOKEN:
-  Installing bison...
-  ```
+##### Install a specific NCRC application version
 
-- Install a specific version
+Don't want to use the latest? For whatever the reason, you can install a specific version of an available NCRC application in the following way:
```
Comment (cticenhour) [inline modules/doc/content/ncrc/ncrc_local_binary.md:95]:
```suggestion
Don't want to use the latest version? For whatever the reason, you can install a specific version of an available NCRC application in the following way:
```

File: modules/doc/content/ncrc/ncrc_local_binary.md [hunk 4]
```diff
@@ -42,144 +42,176 @@ optional arguments:
...
 
-## Uninstalling NCRC Applications
+Username: johndoe
+PIN+TOKEN:
+```
 
-The NCRC script being a wrapper tool, is unable to perform such a function. The user must deactivate the environment and remove that environment using the appropiate conda commands instead:
+##### Uninstalling NCRC application
+
+The NCRC script being a wrapper tool, is unable to perform such a function. The user must deactivate the environment
```
Comment (cticenhour) [inline modules/doc/content/ncrc/ncrc_local_binary.md:121]:
```suggestion
The NCRC script, being a wrapper tool, is unable to perform such a function. The user must deactivate the environment
```

File: modules/doc/content/ncrc/ncrc_local_binary.md [hunk 5]
```diff
@@ -42,144 +42,176 @@ optional arguments:
...
+$ (bison) > cd ~/path/to/my/input/file
+$ (bison) > bison-opt -i my_input.i
+# Using MPI
+$ (bison) > mpiexec -n 8 bison-opt -i my_input.i
+```
+
+##### View Documentation
+
+Documentation exists as part of the binary installation process. You can access documentation by instructing your favorite web browser to open the index.html file.
+We have made finding the location of this file via an environment variable, prefix with the NCRC application name:
```
Comment (GiudGiud) [inline modules/doc/content/ncrc/ncrc_local_binary.md:203]:
```suggestion
We have made finding the location of this file easier via an environment variable, prefixed with the NCRC application name:
```

File: modules/doc/content/ncrc/ncrc_local_binary.md [hunk 6]
```diff
@@ -42,144 +42,176 @@ optional arguments:
...
+$> conda activate bison
+$ (bison) > ncrc update bison
 
-## Uninstalling NCRC Applications
+Username: johndoe
+PIN+TOKEN:
+```
 
-The NCRC script being a wrapper tool, is unable to perform such a function. The user must deactivate the environment and remove that environment using the appropiate conda commands instead:
+##### Uninstalling NCRC application
```
Comment (cticenhour) [inline modules/doc/content/ncrc/ncrc_local_binary.md:119]:
```suggestion
##### Uninstalling an NCRC application
```

File: modules/doc/content/ncrc/ncrc_local_binary.md [hunk 7]
```diff
@@ -42,144 +42,176 @@ optional arguments:
...
-  /home/johndoe/miniconda3/envs/bison/bison/share/bison/doc/index.html
-  ```
-
-  You can then enter that location into your favorite web browser. Similarly on Macintosh machines, you can make use of the `open` command, to open the environment variable:
-
-  ```bash
-  $> conda activate bison
-  $ (bison) > open $bison_DOCS
-  ```
+Where applicable, you only need to activate the environment once. Each example below assume the environment has not yet been activated.
```
Comment (cticenhour) [inline modules/doc/content/ncrc/ncrc_local_binary.md:140]:
```suggestion
Where applicable, you only need to activate the environment once. Each example below assumes the environment has not yet been activated.
```

File: modules/doc/content/ncrc/ncrc_local_binary.md [hunk 8]
```diff
@@ -42,144 +42,176 @@ optional arguments:
...
+--------------------------------------------------------------------------------------------------------------
+Ran 1606 tests in 274.6 seconds. Average test time 0.7 seconds, maximum test time 20.7 seconds.
+1606 passed, 22 skipped, 0 pending, 0 failed
+```
+
+##### Some Applications contain more than tests
+
+Some applications have different inputs that can be copied and executed. Each application can use whatever naming convention
+they wish, so listing all possibilities is impossible. However, any runnable input directory will have a file called `testroot`
+residing in it. The following *crazy* command will return those possibilities (using Bison as an example):
```
Comment (cticenhour) [inline modules/doc/content/ncrc/ncrc_local_binary.md:173]: 🤪

**Review by GiudGiud — COMMENTED** [review]
do you advise to use a different environment for every app?

File: modules/doc/content/ncrc/ncrc_local_binary.md [hunk 9]
```diff
@@ -42,144 +42,176 @@ optional arguments:
...
+$ (bison) >
+$ (bison) > cd ~/path/to/my/input/file
+$ (bison) > bison-opt -i my_input.i
+# Using MPI
+$ (bison) > mpiexec -n 8 bison-opt -i my_input.i
+```
+
+##### View Documentation
+
+Documentation exists as part of the binary installation process. You can access documentation by instructing your favorite web browser to open the index.html file.
```
Comment (cticenhour) [inline modules/doc/content/ncrc/ncrc_local_binary.md:202]:
```suggestion
Documentation exists as part of the binary installation process. You can access documentation by instructing your favorite web browser to open the `index.html` file.
```

File: modules/doc/content/ncrc/ncrc_local_binary.md (hunk 5 above)
Comment (cticenhour) [inline modules/doc/content/ncrc/ncrc_local_binary.md:203]:
```suggestion
We have made finding the location of this file via an environment variable, prefixed with the NCRC application name:
```

**Review by cticenhour — CHANGES_REQUESTED** [review]

File: modules/doc/content/ncrc/ncrc_local_binary.md (hunk 5 above)
Comment (milljm) [reply modules/doc/content/ncrc/ncrc_local_binary.md:203]: Its very much necessary in fact. Griffin for example my need/require a different set of libraries/versions of libMesh at the time the application was built.

File: modules/doc/content/ncrc/ncrc_local_binary.md (hunk 1 above)
Comment (milljm) [reply modules/doc/content/ncrc/ncrc_local_binary.md:53]: That happens to be a comment being generated by the `ncrc` script:

```pre
\```bash
 $> ncrc list

# Use 'ncrc search name-of-application' to list more detail
# NCRC applications available:

pronghorn
bison
griffin
sabertooth
relap7
\```
```

Comment (milljm) [reply modules/doc/content/ncrc/ncrc_local_binary.md:53]: I've been re-doing the indentation going on. I don't much care for the itemized list justification going on, for only half the page...

File: modules/doc/content/ncrc/ncrc_local_binary.md (hunk 8 above)
Comment (milljm) [reply modules/doc/content/ncrc/ncrc_local_binary.md:173]: yeah... hahahaha any help here in making this... better described is appreciated!

File: modules/doc/content/ncrc/ncrc_local_binary.md (hunk 1 above)
Comment (cticenhour) [reply modules/doc/content/ncrc/ncrc_local_binary.md:53]: Oh sorry, I completely missed that it was part of the output..... You can disregard this one. I liked how you moved away from so many itemized lists as well.

File: modules/doc/content/ncrc/ncrc_local_binary.md (hunk 5 above)
Comment (GiudGiud) [reply modules/doc/content/ncrc/ncrc_local_binary.md:203]: ok let's say it explicitly somewhere then? 

unless I missed it

Comment (milljm) [reply modules/doc/content/ncrc/ncrc_local_binary.md:203]: Well, actually I am wrong in how I originally defended the requirement. But they still should be separated. The ncrc assumes the application name is also the environment name.

File: modules/doc/content/ncrc/ncrc_local_binary.md (hunk 8 above)
Comment (cticenhour) [inline modules/doc/content/ncrc/ncrc_local_binary.md:173]:
```suggestion
Some applications have different inputs that can be copied and executed using the `copy-inputs` flag. Each application can use whatever naming convention
they wish for these input categories, so listing all possibilities is impossible. However, any runnable input directory will have a file called `testroot`
residing in it, in which the test categories are listed. The following piped command sequence will return those possibilities (using Bison as an example):
```

Comment (cticenhour) [reply modules/doc/content/ncrc/ncrc_local_binary.md:173]: Added another comment with a wording suggestion that you can take or leave. I appreciate that you are giving a resource for the user to find what those input categories actually are.

Comment (milljm) [reply modules/doc/content/ncrc/ncrc_local_binary.md:173]: Yeah, I then need to add what to actually do with these. Basically another example of how one runs tests.

File: modules/doc/content/ncrc/ncrc_local_binary.md (hunk 5 above)
Comment (milljm) [reply modules/doc/content/ncrc/ncrc_local_binary.md:203]: Yeah I'll make mention of this in the first alert on the page. In my defense, the ncrc script itself will actually warn/error when the user tries to use it while not in the base environment (except during upgrades).

File: framework/doc/content/css/moose.css [hunk 10]
```diff
@@ -610,7 +611,7 @@ footer.page-footer {
 }
 
 .container{
-    width:70%;
+    width:80%;
```
Comment (milljm) [inline framework/doc/content/css/moose.css:614]: I think an additional 10% usable space looks great...

**Review by cticenhour — APPROVED** [review]

File: modules/doc/content/ncrc/ncrc_local_binary.md (hunk 5 above)
Comment (cticenhour) [reply modules/doc/content/ncrc/ncrc_local_binary.md]: More content regarding separate environments for each app was added in subsequent commits. I think this is resolved.

**Pull request merged**